_posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md

```diff
@@ -0,0 +1,139 @@
+
+For more information, check out the [Pulp Import and Export workflow](https://docs.pulpproject.org/pulpcore/en/3.10/workflows/import-export.html) documentation.
+
+## Update to the default Pulp deployment layout
+
+`MEDIA_ROOT` is now its own directory. This change has the advantage of limiting the file permissions and the SELinux file contexts in a shared Pulp 2 and Pulp 3 deployment. This also adds compatibility with the `django_extensions' unreferenced_files` command, which lists all files in `MEDIA_ROOT` that are not in the database.
+
+Other paths are kept on the same absolute paths.
+
+The settings and logging documentation has been updated to reflect these changes. [#7178](https://pulp.plan.io/issues/7178)
+
+## New generic list endpoint
+
+This release includes a new endpoint that you can use to list all `Content`, `ContentGuards`, and `Repositories`. [#7204](https://pulp.plan.io/issues/7204)
+
+## Update to the SigningService
+
+A new `public_key` field has been added to the **SigningService**. This field preserves the value of the public key. A new `pubkey_fingerprint` field was introduced as well, which identifies the
+public key. [#7700](https://pulp.plan.io/issues/7700)
+
+As a result of this, the following deprecation notice has been added for this release:
+
+`The path to the public key is no longer required to be present in the output of a signing script. It is recommended to pass the value of the public key during the object creation process instead.`
+
+## New name filtering options
+
+In previous versions, you could only query the API to search for exact names. With this release, new querying parameters have been added, such as `name__icontains`, `name__contains`, and `name__startswith`, to make searching for objects easier. [#8094](https://pulp.plan.io/issues/8094)
+
+
+## New filter capability for user and group endpoints
+
+You can now filter lists of users and groups.  [#7975](https://pulp.plan.io/issues/7975)
+
+The users endpoint now has the following filter options:         
+
+```
+fields = {
+     "username": ["exact", "iexact", "in", "contains", "icontains"],
+     "first_name": ["exact", "iexact", "in", "contains", "icontains"],
+     "last_name": ["exact", "iexact", "in", "contains", "icontains"],
+     "email": ["exact", "iexact", "in", "contains", "icontains"],
+     "is_active": ["exact"],
+     "is_staff": ["exact"],
+ }
+
+```
+
+The group endpoint now has the following filter options:
+
+```
+fields = {
+         "id": ["exact", "in"],
+         "name": ["exact", "iexact", "in", "contains", "icontains"],
+     }
+```
+
+
+## Add your own labels to objects
+
+With the addition of the `pulp_labels` field, you can create labels to add your own key/data for your resources in Pulp. You can also modify and remove these labels if needed.  
```

Comment:
This section looks good. Can you link to the new doc I created?

https://docs.pulpproject.org/pulpcore/workflows/labels.html

Reply:
Gladly